hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -2338,21 +2344,25 @@ func (r *NodePoolReconciler) listMachineTemplates(nodePool *hyperv1.NodePool) ([
 	switch nodePool.Spec.Platform.Type {
 	// Define the desired template type and mutateTemplate function.
 	case hyperv1.AWSPlatform:
+		capiaws.AddToScheme(api.Scheme)
 		gvk, err = apiutil.GVKForObject(&capiaws.AWSMachineTemplate{}, api.Scheme)
 		if err != nil {
 			return nil, err
 		}
 	case hyperv1.KubevirtPlatform:
+		capikubevirt.AddToScheme(api.Scheme)
```

Comment:
We want to have a single call to AddToScheme for each API that needs to be included, usually this goes within an init().

Reply:
Fixed